part-configuration.tex

```diff
@@ -187,20 +187,245 @@ \section{Matching your business processes}
+\begin{itemize}
+    \item General.xml
+    \item GeneralHierarchical.xml
+    \item Manufacturing.xml
+    \item Service.xml
+    \item UCOA-Form990.xml
+    \item UCOA-Form990EZ.xml
+\end{itemize}
+
+Similar predefined Charts of Accounts are available for other countries and languages.
+
+See \secref{subsec-create-setup-select-coa} for instructions on using these predefined accounts.
+
+The following examples use the \texttt{GeneralHierarchical.xml} account structure.
+This file can be found at \url{https://github.com/ledgersmb/LedgerSMB/blob/master/locale/coa/us/GeneralHierarchical.xml}.
+
+\subsection{Account structure}
+\label{sec-coa-accounts-structure}
+
+The system allows ordering accounts into groups by assigning accounts to headers. Headers
+can themselves be assigned to other headers resulting in hierarchies of account groups.\footnote{Although the
+    database structure supports this type of account hierarchy from  1.3, it was not fully in use in reports until  1.4.17. In 1.3 accounts can be assigned a header,
+    but headers can't be assigned to headers themselves.}
+
+There are multiple reasons for wanting to use hierarchies. One of those reasons is that expense accounts in prior versions were listed as one long list under Expenses while income accounts are listed as a -usually shorter- list under Income. Personnel expenses are usually spread across multiple expense accounts. With lots of other expense accounts also listed, what's the total expense on personnel?
+
+Another use-case for using hierarchies might be because you're running a "margin business". That is, you're only buying what you're selling and the cost of goods sold is the large part of your expenses. In cases like this it can be a good idea to use a different setup of the income statement: one where cost of goods sold is immediately subtracted from income, leaving gross margin to cover all other expenses in the company.
+
+In order to set a heading's parent, go to the \menupath{General Journal \ma Chart of Accounts}. Click on the account number. If the account is a heading account that has a parent heading, the resulting screen shown in \figref{fig:config-coa-hierarchy}.
+
+\begin{figure}[H]
+    \centering
+    \includegraphics[width=\graphicswidth]{images/gl-coa-heading.png}
+    \caption{Configure Account Hierarchy}
+    \label{fig:config-coa-hierarchy}
+\end{figure}
+
+If the account is a not a heading account, but has a parent heading, then the result is shown in \figref{fig:coa-account-setup}.
+
+\begin{figure}[H]
+    \centering
+    \includegraphics[width=\linewidth]{images/gl-coa.png}
+    \caption{Account Setup}
+    \label{fig:coa-account-setup}
+\end{figure}
+
+When creating "Income Statement" \index{income statement} or "Balance Sheet" \index{balance sheet}reports, for example, \menupath{Reports \ma Income Statement} you can select to report using the full hierarchy or just the account category.  See \figref{fig:config-report-hierarchy}

```

Comment:
Note that this option is only available when the "Current Earnings Account" has been set in System > Defaults.

Reply:
Reworded the paragraph to note this.